apps/examples/helloxx/Kconfig

```diff
@@ -6,7 +6,7 @@
 config EXAMPLES_HELLOXX
 	bool "\"Hello, World!\" C++ example"
 	default n
-	depends on HAVE_CXXINITIALIZE
+	depends on HAVE_CXXINITIALIZE || BINFMT_CONSTRUCTORS
```

Comment:
When BINFMT is enabled and HAVE_CXXINIT is disabled, is this working?

Reply:
In the case of flat build, CONFIG_HAVE_CXXINITIALIZE is for calling the static object constructors for kernel

https://github.com/Samsung/TizenRT/blob/8a30134393295de817a8a41de90749489ab44a0f/apps/system/init/init.c#L124-L129

CONFIG_BINFMT_CONSTRUCTORS also does the same thing while loading the application. So they both need not be enabled for loadable as we have cpp only in the app side.

https://github.com/Samsung/TizenRT/blob/3fc8caed54420a3dfa6e002e8807feee20407142/os/binfmt/binfmt_execmodule.c#L116-L132